### ISSUE TITLE
Question Windows 11 Latest Photoshop and Premier Pro

### ISSUE THREAD

**Issue by megvadulthangya** [pal1000/mesa-dist-win#151]
Hi!
Im wondering this project can help to resolve my problems...

I have outdated configs, and none of them can run the latest photoshop couse its needs natively supported DX12 gpu...

Laptop:

Dell E6330 i5 3360M cpu with HD4000 => This config cant run none of the mentioned Programs with GPU acceleration

Desktop
i7-7700K with GTX 750 Ti 4GB => This config can run latest premier pro without a problem, but photoshop if i enable the GPU acceleration i get only a white flickering screen when i open a picture....

Please advise and if i wrote to a bad place please direct me to (mail, discord, or forum) where can i ask, this and how to do correctly if its possible...

Thank you for your time.

**Comment by pal1000** [comment]
Yes, this project may help. You should try it on i7-7700K system as it's far more powerful.
